src/components/tools/table-tool/rename-column-dialog.tsx

```diff
@@ -9,7 +9,7 @@ interface IProps {
   onRenameAttribute: (id: string, name: string) => void;
   onClose: () => void;
   name: string;
-  nameValidator?: (name: string) => boolean;
+  columnNameErrorGetter?: (name: string) => string | undefined;
```

Comment:
I think you can still call it the `columnNameValidator` (which sounds better than `columnNameErrorGetter`) even though it returns an error string rather than a boolean.